webroot/js/components/platform-logos-list.js

```diff
@@ -1,10 +1,52 @@
-import { h, Component } from '/js/web_modules/preact.js';
+import {h} from '/js/web_modules/preact.js';
 import htm from '/js/web_modules/htm.js';
+import {SOCIAL_PLATFORMS} from '../utils/platforms.js';
+import {classNames} from '../utils/helpers.js';
+
 const html = htm.bind(h);
 
-import { SOCIAL_PLATFORMS } from '../utils/platforms.js';
-import { classNames } from '../utils/helpers.js';
+function SocialIcon(props) {
+  const {platform, icon, url, color} = props;
+  const platformInfo = SOCIAL_PLATFORMS[platform.toLowerCase()];
+  const inList = !!platformInfo;
+  const iconSupplied = !!icon
+  const name = inList ? platformInfo.name : platform;
+  const colorSupplied = !!color
+
+  const finalIcon = iconSupplied ? icon : (inList ? platformInfo.icon : '/img/socialhandles/default.svg')
+  const finalColor = colorSupplied ? color : (inList ? platformInfo?.color ? platformInfo.color : '#000000' : '#000000');
+
+  const style = `-webkit-mask-image: ${finalIcon}; mask-image: url(${finalIcon}); background-color: ${finalColor};`
```

Comment:
Hey @SeaLife I tried checking out your SVG icons, but I couldn't get them to show up. For some reason the `mask-image` styles aren't getting rendered for me.

<img width="1195" alt="image" src="https://user-images.githubusercontent.com/9563255/103434058-a6d16480-4bb0-11eb-938f-576170072990.png">